sklearn/pipeline.py

```diff
@@ -787,6 +787,23 @@ def make_union(*transformers, **kwargs):
     and does not permit, naming the transformers. Instead, they will be given
     names automatically based on their types. It also does not allow weighting.
 
+    Parameters
+    ----------
+    transformer_list : list of (string, transformer) tuples
```

Comment:
This is not correct. You want something like. `*transformers : list of estimators`